src/Arbiter.tsx

```diff
@@ -139,65 +137,55 @@ export const Arbiter = ({ NFToupon_Key }: Props) => {
         }
       };
     }
-  }, [xummPayload, NFToupon_Key]);
+  }, [xummPayload,
+    NFToupon_Key,
+  ]);
 
   useEffect(() => {
     if (transactionType === 'NFTokenCreateOffer') {
       const update = async () => {
-        await fetch('https://eatozee-crypto.app/api/nftoupon/merchant/update', {
-          method: 'POST',
-          headers: {
-            'Content-Type': 'application/json',
-            'NFToupon-Key': NFToupon_Key,
-          },
-          body: JSON.stringify({
-            id: details.id,
-            status: sendProperties.status,
-            date: sendProperties.expiryDate,
-            offer: sendProperties.offer,
-            merchantCryptoWalletAddress: walletAddress,
-            transactionType,
-            tokenId: details.tokenId,
-          }),
-        });
+        const payload = {
+          id: details.id,
+          status: sendProperties.status,
+          date: sendProperties.expiryDate,
+          offer: sendProperties.offer,
+          merchantCryptoWalletAddress: walletAddress,
+          transactionType,
+          tokenId: details.tokenId,
+        };
+
+        await fetcher(NFToupon_Key, MERCHANT_OFFER_URL, payload);
```

Comment:
I assume @devanpatel1008 will add the toast if there is a api failure .. @dhruvin2803 is that correct?